mods/persistence/modules/projectiles/overrides.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/gun/launcher/syringe/rapid
+	origin_tech = "{'wormholes':1337}"
```

Comment:
Setting origin tech like this should be unnecessary. Instead set ``research_excluded = TRUE``

Reply:
done